client/my-sites/upgrades/domain-management/name-servers/dns-template-button.jsx

```diff
@@ -1,14 +1,14 @@
 /**
  * External dependencies
  */
-import React, { Component } from 'react';
+import React from 'react';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 
-class DnsTemplateButton extends Component {
+class DnsTemplateButton extends Button {
```

Comment:
Sometimes we're stuck in a loop and need outside perspective to see the solution - glad I could help 😄